package.json

```diff
@@ -28,39 +28,54 @@
   "bugs": {
     "url": "https://github.com/aerospike/aerospike-client-nodejs/issues"
   },
+  "binary": {
+    "module_name": "aerospike",
+    "module_path": "./lib/binding/{configuration}/{node_abi}-{platform}-{arch}/",
+    "remote_path": "./{module_name}/v{version}/{configuration}/",
+    "package_name": "{module_name}-v{version}-{node_abi}-{platform}-{arch}.tar.gz",
+    "host": "https://www.npmjs.com/package/aerospike"
+  },
+  "config": {
+    "unsafe-perm": true
+  },
   "scripts": {
-    "test": "mocha",
+    "preinstall": "npm install @mapbox/node-pre-gyp",
+    "install": "node-pre-gyp install --fallback-to-build",
+    "test": "mocha --exit",
```

Comment:
If you have to use `--exit` that strongly indicates that there's either a bug in the client or a problematic test which prevent the event loop from shutting down? We should find and fix those issues. Running mocha with `--exit` just hides the problem.